From 3fe1ea74a50b7815816157da93c60379e8328821 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Mon, 7 Jul 2014 14:58:36 +0200
Subject: [PATCH] man: chroot jails are no longer detected by
 ConditionVirtualization=

(cherry picked from commit ac8ddf8c964f813464ef32cad1fcb7b61b692a01)
---
 man/systemd.unit.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/man/systemd.unit.xml b/man/systemd.unit.xml
index be0873c344..cd3279c192 100644
--- a/man/systemd.unit.xml
+++ b/man/systemd.unit.xml
@@ -990,7 +990,6 @@
                                 <varname>oracle</varname>,
                                 <varname>xen</varname>,
                                 <varname>bochs</varname>,
-                                <varname>chroot</varname>,
                                 <varname>uml</varname>,
                                 <varname>openvz</varname>,
                                 <varname>lxc</varname>,
